integration-tests/instructions-with-bytes.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { CONFIGS } from "./config";
+import { Protocols, TezosOperationError } from "@taquito/taquito";
+import { addContract, lslContract, lsrContract, notContract, orContract, xorContract } from "./data/instructions-with-bytes-contracts";
+
+CONFIGS().forEach(({ lib, protocol, setup }) => {
+  const Tezos = lib;
+  const limanet = protocol === Protocols.PtLimaPtL ? test : test.skip;
+  const mumbaiAndAlpha = protocol === Protocols.PtMumbaii || protocol === Protocols.ProtoALpha ? test : test.skip;
+
+  describe(`Test origination of contract with instructions now supporting bytes`, () => {
+
+    beforeEach(async (done) => {
+      await setup();
+      done();
+    });
+
+    mumbaiAndAlpha(`Should be able to orignate contract with ADD parameter in michelson contract with bytes`, async done => {
+      const contract = await Tezos.contract.originate({
+        code: addContract,
+        storage: 0
+      });
+      await contract.confirmation();
+      expect(contract).toBeDefined();
```

Comment:
could you also add an assertion that the operation status is 'applied' for these tests. Looks good otherwise.

Reply:
👍